### PR TITLE
ValidCertificadora.VIDaaSConnect version 1.0.2

### PR DESCRIPTION
**Pull request by TheKoakuma**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/98548)

### CODE REVIEW

**Comment by TheKoakuma** [comment]
@microsoft-github-policy-service agree

**Comment by TheKoakuma** [comment]
@microsoft-github-policy-service agree

**Comment by TheKoakuma** [comment]
Tried with /silent and /verysilent, but the installer downloads and install Windows Desktop Runtime (id Microsoft.DotNet.DesktopRuntime.3_1) requesting user click on next in both cases.

Should be fixed if dependency is installed before the package

**Comment by stephengillie** [comment]
Hi @TheKoakuma,

The package installs normally, but requires `Microsoft.DotNet.DesktopRuntime.3_1` to run:
image

Pending:
* https://github.com/microsoft/winget-cli/issues/163

**Comment by stephengillie** [comment]
> 12/27/2023 8:45:54 PM
Installer Verification Analysis Context Information: 
Specified hash doesn't match.
Actual hash:   2708FAB120644E3E405EB9B02B494B7B9B80DFF516AAF3A00667AEB50D72001C
Expected hash: 46d069f9a6e80320095c4ed22d5c49115b0118efe7e747fa38daaec6d3d3ede0

File: manifests/v/ValidCertificadora/VIDaaSConnect/1.0.2/ValidCertificadora.VIDaaSConnect.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: ValidCertificadora.VIDaaSConnect
+PackageVersion: 1.0.2
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://certificado-em-nuvem.validcertificadora.com.br/downloads/vidaas_connect_windows10.exe
+  InstallerSha256: FA9D77EF1DE04D2914572E83EBFA1CC96436243C1095E181291E4045540AAD23
```
Comment (stephengillie) [inline manifests/v/ValidCertificadora/VIDaaSConnect/1.0.2/ValidCertificadora.VIDaaSConnect.installer.yaml:10]:
```suggestion
  InstallerSha256: 2708FAB120644E3E405EB9B02B494B7B9B80DFF516AAF3A00667AEB50D72001C
```

Comment (TheKoakuma) [reply manifests/v/ValidCertificadora/VIDaaSConnect/1.0.2/ValidCertificadora.VIDaaSConnect.installer.yaml:10]: Lgfm

**Review by Trenly — CHANGES_REQUESTED** [review]
The URL for this package now points to version 1.0.4. Please create a new PR for the latest version

**Comment by TheKoakuma** [comment]
Closing due Outdated version (Program and Manifest) 

Updated PR #132444

**Pull request closed without merging**